packages/open-truss/src/pages/config-builder-page/index.tsx

```diff
@@ -12,6 +16,24 @@ interface ConfigBuilderPageInterface {
   components: COMPONENTS
 }
 
+const Wrapper: FrameWrapper = ({ children, configPath, frame }) => {
```

Comment:
The eslint rule change I made makes declaring a function like this possible. Before you'd have to have a type for each prop defined but they're already defined in `FrameWrapper`

Reply:
I'll clean up other declarations like this if we're into it!

Reply:
oh rad, that's way cleaner! yes please!